x-pack/plugins/transform/public/app/common/validators/integer_range_10_to_10000_validator.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { numberValidator } from '@kbn/ml-agg-utils';
+
+import type { Validator } from './types';
+
+const numberRange10To10000NotValidErrorMessage = i18n.translate(
+  'xpack.transform.transformSettingValidations.numberRange10To10000NotValidErrorMessage',
+  {
+    defaultMessage: 'Value needs to be an integer between 10 and 10000.',
+  }
+);
+
+export const integerRange10To10000Validator: Validator = (value) =>
```

Comment:
This function seems too specific to me. Should we create a generic validator for string representation of numbers? Otherwise, we end up with loads of similar functions for each specific case.

Reply:
Looks like this was no longer in use anyway, so I removed it in [90e20ac](https://github.com/elastic/kibana/pull/173736/commits/90e20ac7f6038e4243a0645384e1ee88ffecbb8f).